## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 427212`
Commit: `1cea994d2be7fb016b77d13546dd0337eaf99ab2`

---
### `aarch64-darwin`
<details>
  <summary>:x: 34 packages failed to build:</summary>
  <ul>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 104 packages built:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.ormsgpack</li>
    <li>python312Packages.ormsgpack.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-sdk</li>
    <li>python313Packages.langgraph-sdk.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.ormsgpack</li>
    <li>python313Packages.ormsgpack.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.langchain-anthropic</summary>
<pre> - Adding langchain_anthropic/__init__.py -> langchain_anthropic/__init__.py
 - Adding langchain_anthropic/_client_utils.py -> langchain_anthropic/_client_utils.py
 - Adding langchain_anthropic/chat_models.py -> langchain_anthropic/chat_models.py
 - Adding langchain_anthropic/experimental.py -> langchain_anthropic/experimental.py
 - Adding langchain_anthropic/llms.py -> langchain_anthropic/llms.py
 - Adding langchain_anthropic/output_parsers.py -> langchain_anthropic/output_parsers.py
 - Adding langchain_anthropic/py.typed -> langchain_anthropic/py.typed
 - Adding langchain_anthropic-0.3.17.dist-info/RECORD -> langchain_anthropic-0.3.17.dist-info/RECORD
Successfully built langchain_anthropic-0.3.17-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-python3.12-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic/dist /private/tmp/nix-build-python3.12-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Unpacking to: unpacked/langchain_anthropic-0.3.17...OK
Repacking wheel as ./langchain_anthropic-0.3.17-py3-none-any.whl...OK
/private/tmp/nix-build-python3.12-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for langchain_anthropic-0.3.17-py3-none-any.whl
  - anthropic<1,>=0.57.0 not satisfied by version 0.55.0</pre>
</details>
<details>
<summary>python312Packages.langchain-openai</summary>
<pre>benchmark: 5.1.0 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /private/tmp/nix-build-python3.12-langchain-openai-0.3.28.drv-0/source/libs/partners/openai
configfile: pyproject.toml
plugins: anyio-4.9.0, langsmith-0.3.45, asyncio-0.26.0, benchmark-5.1.0, codspeed-3.2.0, recording-0.13.4, socket-0.7.0, syrupy-4.9.1, cov-nixpkgs-stub-1.0.0, mock-3.14.1, requests-mock-1.12.1
asyncio: mode=Mode.AUTO, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... 
collected 237 items / 1 error / 37 deselected / 200 selected                   

==================================== ERRORS ====================================
____ ERROR collecting tests/unit_tests/chat_models/test_responses_stream.py ____
tests/unit_tests/chat_models/test_responses_stream.py:234: in <module>
    ResponseTextDeltaEvent(
E   pydantic_core._pydantic_core.ValidationError: 1 validation error for ResponseTextDeltaEvent
E   logprobs
E     Field required [type=missing, input_value={'content_index': 0, 'del...onse.output_text.delta'}, input_type=dict]
E       For further information visit https://errors.pydantic.dev/2.11/v/missing
=========================== short test summary info ============================
ERROR tests/unit_tests/chat_models/test_responses_stream.py - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 37 deselected, 1 error in 0.56s ========================</pre>
</details>
<details>
<summary>python313Packages.langchain-anthropic</summary>
<pre> - Adding langchain_anthropic/__init__.py -> langchain_anthropic/__init__.py
 - Adding langchain_anthropic/_client_utils.py -> langchain_anthropic/_client_utils.py
 - Adding langchain_anthropic/chat_models.py -> langchain_anthropic/chat_models.py
 - Adding langchain_anthropic/experimental.py -> langchain_anthropic/experimental.py
 - Adding langchain_anthropic/llms.py -> langchain_anthropic/llms.py
 - Adding langchain_anthropic/output_parsers.py -> langchain_anthropic/output_parsers.py
 - Adding langchain_anthropic/py.typed -> langchain_anthropic/py.typed
 - Adding langchain_anthropic-0.3.17.dist-info/RECORD -> langchain_anthropic-0.3.17.dist-info/RECORD
Successfully built langchain_anthropic-0.3.17-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-python3.13-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic/dist /private/tmp/nix-build-python3.13-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Unpacking to: unpacked/langchain_anthropic-0.3.17...OK
Repacking wheel as ./langchain_anthropic-0.3.17-py3-none-any.whl...OK
/private/tmp/nix-build-python3.13-langchain-anthropic-0.3.17.drv-0/source/libs/partners/anthropic
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for langchain_anthropic-0.3.17-py3-none-any.whl
  - anthropic<1,>=0.57.0 not satisfied by version 0.55.0</pre>
</details>
<details>
<summary>python313Packages.langchain-openai</summary>
<pre>configfile: pyproject.toml
plugins: anyio-4.9.0, langsmith-0.3.45, asyncio-0.26.0, benchmark-5.1.0, codspeed-3.2.0, recording-0.13.4, socket-0.7.0, syrupy-4.9.1, cov-nixpkgs-stub-1.0.0, mock-3.14.1, requests-mock-1.12.1
asyncio: mode=Mode.AUTO, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... 
collecting 5 items                                                             
collecting 177 items / 1 error                                                 
collected 237 items / 1 error / 37 deselected / 200 selected                   

==================================== ERRORS ====================================
____ ERROR collecting tests/unit_tests/chat_models/test_responses_stream.py ____
tests/unit_tests/chat_models/test_responses_stream.py:234: in <module>
    ResponseTextDeltaEvent(
E   pydantic_core._pydantic_core.ValidationError: 1 validation error for ResponseTextDeltaEvent
E   logprobs
E     Field required [type=missing, input_value={'content_index': 0, 'del...onse.output_text.delta'}, input_type=dict]
E       For further information visit https://errors.pydantic.dev/2.11/v/missing
=========================== short test summary info ============================
ERROR tests/unit_tests/chat_models/test_responses_stream.py - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 37 deselected, 1 error in 1.33s ========================</pre>
</details>
</details>

--- a/pkgs/development/python-modules/langchain-anthropic/default.nix
+++ b/pkgs/development/python-modules/langchain-anthropic/default.nix
@@ -22,14 +22,14 @@
 
 buildPythonPackage rec {
   pname = "langchain-anthropic";
-  version = "0.3.15";
+  version = "0.3.17";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
     tag = "langchain-anthropic==${version}";
-    hash = "sha256-GOD6pMuUDCfrQ6MP+/HXZIg5wHUDRysosEjXjewY/9M=";
+    hash = "sha256-oUT4Mu/vG+bVF6zLQX2RbVUglJ6VMyBt8XtCBSlBlpU=";
   };
 
   sourceRoot = "${src.name}/libs/partners/anthropic";

--- a/pkgs/development/python-modules/langgraph-checkpoint-sqlite/default.nix
+++ b/pkgs/development/python-modules/langgraph-checkpoint-sqlite/default.nix
@@ -4,15 +4,17 @@
   fetchFromGitHub,
 
   # build system
-  poetry-core,
+  hatchling,
 
   # dependencies
   aiosqlite,
   langgraph-checkpoint,
+  sqlite-vec,
 
   # testing
   pytest-asyncio,
   pytestCheckHook,
+  sqlite,
 
   # passthru
   gitUpdater,
@@ -20,27 +22,32 @@
 
 buildPythonPackage rec {
   pname = "langgraph-checkpoint-sqlite";
-  version = "2.0.6";
+  version = "2.0.10";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langgraph";
     tag = "checkpointsqlite==${version}";
-    hash = "sha256-UUlrhQS0C2rPp//+LwU2rgR4R3AM5fM9X3CYvi/DAy8=";
+    hash = "sha256-570rXAxS4p2b7fc1aqSNArXHTz7G2GjZQDqMlqK7Jso=";
   };
 
   sourceRoot = "${src.name}/libs/checkpoint-sqlite";
 
-  build-system = [ poetry-core ];
+  build-system = [ hatchling ];
 
   dependencies = [
     aiosqlite
     langgraph-checkpoint
+    sqlite-vec
   ];
 
   pythonRelaxDeps = [
     "aiosqlite"
+
+    # Bug: version is showing up as 0.0.0
+    # https://github.com/NixOS/nixpkgs/issues/427197
+    "sqlite-vec"
 
     # Checkpoint clients are lagging behind langgraph-checkpoint
     "langgraph-checkpoint"
@@ -51,6 +58,20 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     pytest-asyncio
     pytestCheckHook
+    sqlite
+  ];
+
+  disabledTestPaths = [
+    # Failed: 'flaky' not found in `markers` configuration option
+    "tests/test_ttl.py"
+  ];
+
+  disabledTests = [
+    # AssertionError: (fails object comparison due to extra runtime fields)
+    # https://github.com/langchain-ai/langgraph/issues/5604
+    "test_combined_metadata"
+    "test_asearch"
+    "test_search"
   ];
 
   passthru.updateScript = gitUpdater {

--- a/pkgs/development/python-modules/langgraph-sdk/default.nix
+++ b/pkgs/development/python-modules/langgraph-sdk/default.nix
@@ -4,7 +4,7 @@
   fetchFromGitHub,
 
   # build-system
-  poetry-core,
+  hatchling,
 
   # dependencies
   httpx,
@@ -18,19 +18,19 @@
 
 buildPythonPackage rec {
   pname = "langgraph-sdk";
-  version = "0.1.69";
+  version = "0.2.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langgraph";
     tag = "sdk==${version}";
-    hash = "sha256-MRs5crbUEak/fr17+lerGFY+xTm7sanUW1lZXbPBAeg=";
+    hash = "sha256-uhVdtB/fLy0hfZKfzNV2eoO83bvKppGVl4Lm8IEscL0=";
   };
 
   sourceRoot = "${src.name}/libs/sdk-py";
 
-  build-system = [ poetry-core ];
+  build-system = [ hatchling ];
 
   dependencies = [
     httpx

